RecoBTag/SoftLepton/src/MuonTagger.cc

```diff
@@ -35,8 +36,8 @@ float MuonTagger::discriminator(const TagInfoHelper& tagInfo) const {
     bool flip(false);
     if(m_selector.isNegative()) {
       int seed=1+round(10000.*properties.deltaR);
```

Comment:
My understanding is all random numbers used in RECO are seeded deterministically. The reason is to guarantee we get the same results each time we run the RECO.